### PR TITLE
Improve website

### PR DESCRIPTION
**Pull request by itexpert120**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -200,30 +236,149 @@ function App(props) {
...
+            <Route path={"/scanner"}>
+              <NavigationWrapper {...passProps} />
+              <KeypomScanner />
+            </Route>
+            <Route path={"/create"}>
+              <ViewPage overrideSrc={passProps.widgets.create} {...passProps} />
+              <Footer {...passProps} />
+            </Route>
+            {routes.map((route) => (
+              <Router key={`${route.path}`} path={route.path}>
```
Comment (elliotBraem) [inline src/App.js]: Wait... are you sure this should be a nested Router?
Shouldn't this be Route

File: public/template.html
Comment (elliotBraem) [inline public/template.html:1]: This was renamed -- do you need to update _redirects? 

Also, should change Title, Description
CNAME
Update the manifest.json

File: src/App.js (hunk 1 above)
Comment (itexpert120) [reply src/App.js]: My mistake, I corrected it

File: public/template.html
Comment (itexpert120) [reply public/template.html:1]: resolved

**Pull request merged**